jobbergate-cli/jobbergate_cli/config.py

```diff
@@ -93,22 +79,31 @@ def compute_extra_settings(self) -> Self:
         cache_dir = self.JOBBERGATE_CACHE_DIR
         cache_dir.mkdir(exist_ok=True, parents=True)
 
-        self.JOBBERGATE_APPLICATION_MODULE_PATH = cache_dir / constants.JOBBERGATE_APPLICATION_MODULE_FILE_NAME
-        self.JOBBERGATE_APPLICATION_CONFIG_PATH = cache_dir / constants.JOBBERGATE_APPLICATION_CONFIG_FILE_NAME
-
-        log_dir = cache_dir / "logs"
-        log_dir.mkdir(exist_ok=True, parents=True)
-        self.JOBBERGATE_LOG_PATH = log_dir / "jobbergate-cli.log"
+        return self
```

Comment:
This method needs to have it's docstring update to reflect what it's actually doing now.

Reply:
Good catch.

Since just `JOBBERGATE_LOG_PATH` was left, I changed it to a field validator for clarification. Solved in f706334.